packages/jest-jasmine2/src/jasmine/Env.ts

```diff
@@ -0,0 +1,665 @@
+            'Tests cannot be nested. Test `' +
+              spec.description +
+              '` cannot run because it is nested within `' +
+              currentSpec.description +
+              '`.',
+          );
+        }
+        currentDeclarationSuite.addChild(spec);
+        return spec;
+      };
+
+      this.xit = function(...args) {
+        const spec = this.it.apply(this, args);
+        spec.pend('Temporarily disabled with xit');
+        return spec;
+      };
+
+      this.todo = function() {
+        const description = arguments[0];
+        if (arguments.length !== 1 || typeof description !== 'string') {
+          throw new ErrorWithStack(
+            'Todo must be called with only a description.',
+            test.todo,
+          );
+        }
+
+        const spec = specFactory(
+          description,
+          () => {},
+          currentDeclarationSuite,
+        );
+        spec.todo();
+        currentDeclarationSuite.addChild(spec);
+        return spec;
+      };
+
+      this.fit = function(description, fn, timeout) {
+        const spec = specFactory(
+          description,
+          fn,
+          currentDeclarationSuite,
+          timeout,
+        );
+        currentDeclarationSuite.addChild(spec);
+        focusedRunnables.push(spec.id);
+        unfocusAncestor();
+        return spec;
+      };
+
+      this.beforeEach = function(beforeEachFunction, timeout) {
+        currentDeclarationSuite.beforeEach({
+          fn: beforeEachFunction,
+          timeout() {
+            return timeout || j$._DEFAULT_TIMEOUT_INTERVAL;
+          },
+        });
+      };
+
+      this.beforeAll = function(beforeAllFunction, timeout) {
+        currentDeclarationSuite.beforeAll({
+          fn: beforeAllFunction,
+          timeout() {
+            return timeout || j$._DEFAULT_TIMEOUT_INTERVAL;
+          },
+        });
+      };
+
+      this.afterEach = function(afterEachFunction, timeout) {
+        currentDeclarationSuite.afterEach({
+          fn: afterEachFunction,
+          timeout() {
+            return timeout || j$._DEFAULT_TIMEOUT_INTERVAL;
+          },
+        });
+      };
+
+      this.afterAll = function(afterAllFunction, timeout) {
+        currentDeclarationSuite.afterAll({
+          fn: afterAllFunction,
+          timeout() {
+            return timeout || j$._DEFAULT_TIMEOUT_INTERVAL;
+          },
+        });
+      };
+
+      this.pending = function(message) {
+        let fullMessage = j$.Spec.pendingSpecExceptionMessage;
+        if (message) {
+          fullMessage += message;
+        }
+        throw fullMessage;
+      };
+
+      this.fail = function(error) {
+        let checkIsError;
+        let message;
+
+        if (error instanceof AssertionError) {
+          checkIsError = false;
+          // @ts-ignore TODO Possible error: j$.Spec does not have expand property
+          message = assertionErrorMessage(error, {expand: j$.Spec.expand});
```

Comment:
This is also might be error.

Reply:
However tests are passing

Reply:
Since the tests are passing, I wouldn't worry too much about it